src/Compilers/VisualBasic/Portable/Lowering/LambdaRewriter/LambdaRewriter.vb

```diff
@@ -225,54 +225,59 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                     Continue For
                 End If
 
-                Dim frame As LambdaFrame = GetFrameForScope(copyConstructor, captured, node, scopeOrdinalDispenser, _delegateRelaxationIdDispenser)
+                Dim frame As LambdaFrame = GetFrameForScope(copyConstructor, captured, node, closureDebugInfo, _delegateRelaxationIdDispenser)
 
-                Dim proxy = LambdaCapturedVariable.Create(frame, captured, synthesizedFieldNameIdDispenser)
+                Dim proxy = LambdaCapturedVariable.Create(frame, captured, _synthesizedFieldNameIdDispenser)
                 Proxies.Add(captured, proxy)
-                frame.m_captured_locals.Add(proxy)
+                frame.CapturedLocals.Add(proxy)
             Next
 
-            For Each frame In frames.Values
+            For Each frame In _frames.Values
                 CompilationState.AddSynthesizedMethod(frame.Constructor, MakeFrameCtor(frame, Diagnostics))
             Next
         End Sub
 
         Private Function GetFrameForScope(copyConstructor As Boolean,
                                           captured As Symbol,
-                                          node As BoundNode,
-                                          ByRef scopeOrdinalDispenser As Integer,
+                                          scope As BoundNode,
+                                          closureDebugInfo As ArrayBuilder(Of ClosureDebugInfo),
                                           ByRef delegateRelaxationIdDispenser As Integer) As LambdaFrame
             Dim frame As LambdaFrame = Nothing
```

Comment:
It is to avoid warning saying "passed by reference before it has been assigned a value" when calling TryGetValue(scope, frame) at 247.